atlassian_jwt_auth/contrib/tests/utils.py

```diff
@@ -1,20 +1,21 @@
 import atlassian_jwt_auth
 
 
-class StaticPublicKeyRetriever(object):
```

Comment:
I'm glad you asked. The main point of this PR is to expose a setting that is obvious and easy to access during test to allow tests to substitute a public key retriever during tests. Previously, this was very challenging without using `mock.patch()` on some internal implementation detail of the library.

The only problem with my solution is that it expects the user to provide a concrete class that can be instantiated by the Flask plugin to serve as the public key retriever. I wanted to be able to pass keys into that class though and the best way to do that was to create a factory function that would create an anonymous class that contained the static keys that I wanted to use.